plugin/src/main/cpp/include/export/pico_export_plugin.h

```diff
@@ -65,9 +69,11 @@ class PicoEditorExportPlugin : public OpenXREditorExportPlugin {
 
 	Dictionary _eye_tracking_option;
 	Dictionary _face_tracking_option;
+	Dictionary _hand_tracking_option;
 
 private:
 	bool _is_eye_tracking_enabled() const;
+	bool _is_hand_tracking_enabled() const;
```

Comment:
I don't see an implementation for this method.

Reply:
Oops, I forgot about that, thank you! I removed it